### DISCUSSION TITLE
Updating to node-datachannel lastest version

### DISCUSSION THREAD

**Discussion by ghost** [geckosio/geckos.io#256] [General]
Hey @yandeu , can you update to node-datachannel latest version?

Also question if you left the project, because you don't seem to very active there

**Comment by yandeu** [comment]
I will. No, I personally don't use it at the moment, but I'm open for PR's and suggestions and will try to keep it maintained.

_Also, I have a new job since February (as a teacher) and am currently on vacation (1.April - 23.April)._

**Comment by ghost** [comment]
Thanks. Oh that's nice job without being indiscreet what do you teach exactly?
